src/pages/invoices/common/hooks/useResolveInputField.tsx

```diff
@@ -98,7 +98,11 @@ export function useResolveInputField(props: Props) {
               );
           }}
           className="w-36"
-          formatLabel={(resource) => resource.name}
+          formatLabel={(resource) => (
+            <div className="flex justify-between items-center">
+              <span>{resource.name}</span> <span>{resource.rate}%</span>
+            </div>
```

Comment:
Looks good, but the style is off. Please just use the following format: NAME (VALUE %), eg. VAT (15%).